packages/web3-plugin-example/package.json

```diff
@@ -0,0 +1,63 @@
+{
+	"name": "web3-plugin-example",
+	"version": "0.0.1-alpha.0",
+	"description": "",
+	"main": "dist/index.js",
+	"repository": "https://github.com/ChainSafe/web3.js",
+	"engines": {
+		"node": ">=12.0.0"
+	},
+	"author": "ChainSafe Systems",
+	"license": "LGPL-3.0",
+	"keywords": [
+		"Ethereum",
+		"JavaScript",
+		"API"
+	],
+	"files": [
+		"dist/**/*"
+	],
+	"scripts": {
+		"clean": "rimraf dist",
+		"prebuild": "rimraf dist",
+		"build": "tsc --build",
+		"build:web": "npx webpack",
+		"build:check": "node -e \"require('./dist')\"",
+		"lint": "eslint --ext .js,.ts .",
+		"lint:fix": "eslint --fix --ext .js,.ts .",
+		"format": "prettier --write '**/*'",
+		"test": "jest --config=./test/unit/jest.config.js",
+		"test:coverage": "jest --config=./test/unit/jest.config.js --coverage=true --coverage-reporters=text",
+		"test:ci": "jest --coverage=true --coverage-reporters=json --verbose",
+		"test:watch": "npm test -- --watch",
+		"test:unit": "jest --config=./test/unit/jest.config.js",
+		"test:integration": "jest --config=./test/integration/jest.config.js --runInBand --forceExit",
+		"test:blackbox:ganache:http": "./scripts/black_box_test.sh ganache http",
+		"test:blackbox:ganache:ws": "./scripts/black_box_test.sh ganache ws",
+		"test:blackbox:geth:http": "./scripts/black_box_test.sh geth http",
+		"test:blackbox:geth:ws": "./scripts/black_box_test.sh geth ws",
+		"test:blackbox:infura:http": "./scripts/black_box_test.sh infura http",
+		"test:blackbox:infura:ws": "./scripts/black_box_test.sh infura ws"
```

Comment:
I think these are not required in example plugin dir as each plugin might have its own set of tests.

`"test:integration": "jest --config=./test/integration/jest.config.js --runInBand --forceExit",
		"test:blackbox:ganache:http": "./scripts/black_box_test.sh ganache http",
		"test:blackbox:ganache:ws": "./scripts/black_box_test.sh ganache ws",
		"test:blackbox:geth:http": "./scripts/black_box_test.sh geth http",
		"test:blackbox:geth:ws": "./scripts/black_box_test.sh geth ws",
		"test:blackbox:infura:http": "./scripts/black_box_test.sh infura http",
		"test:blackbox:infura:ws": "./scripts/black_box_test.sh infura ws"`

Reply:
Remove in [this](https://github.com/ChainSafe/web3.js/pull/5393/commits/6514f1352645cb6793e9670fe7968c95176e5713) commit